modules/quality_control/php/NDB_Menu_Filter_quality_control_behavioral.class.inc

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * This file contains the NDB_Menu_Filter_quality_control class
+ *
+ * PHP Version 5
+ *
+ * @category Module
+ * @package  Quality_Control
+ * @author   Liza Levitis <llevitis.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/CCNA/
+ */
+/**
+ * Behavioural QC Class
+ *
+ * PHP Version 5
+ *
+ * @category Module
+ * @package  Quality_Control
+ * @author   Liza Levitis <llevitis.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/CCNA/
+ */
+class NDB_Menu_Filter_Quality_Control_Behavioral extends NDB_Menu_Filter
+{
+    public $hasWritePermission = false;
+    public $AjaxModule         = true;
+    /**
+     * Check user permissions
+     *
+     * @return bool
+     * @throws ConfigurationException
+     */
+    function _hasAccess()
+    {
+        //create user object
+        $user =& User::singleton();
+        return $user->hasPermission('data_team_helper');
+    }
+    /**
+     * Override default behaviour, since the page is
+     * loaded from qualityControlIndex.js
+     *
+     * @return void
+     * @access public
+     */
+    function display()
+    {
+        return null;
+    }
```

Comment:
This shouldn't be here. This was a hack that Alex used once, which was replaced by the `$skipTemplate` class variable